Header header

float32[12] cur_angle
float32[12] cur_vel
float32[12] cur_torque
float32 cur_euler_x
float32 cur_euler_y
float32 cur_euler_z